### PR TITLE
:bug: Fix comments and history toggle

### PR DESCRIPTION
**Pull request by AzazelN28**
[Taiga Issue #7242](https://tree.taiga.io/project/penpot/issue/7242) History panel doesn't close when click on comments icon

### CODE REVIEW

File: frontend/src/app/main/data/workspace/drawing.cljs [hunk 1]
```diff
@@ -27,6 +27,8 @@
   (ptk/reify ::select-for-drawing
     ptk/UpdateEvent
     (update [_ state]
+      (when (= tool :comments)
+        (update state :workspace-layout disj :document-history))
```
Comment (niwinz) [inline frontend/src/app/main/data/workspace/drawing.cljs]: This code does nothing, I mean, it returns a new state but this state is ignored.

File: frontend/src/app/main/ui/workspace/right_header.cljs [hunk 2]
```diff
@@ -169,33 +169,32 @@
...
-           (when (contains? layout :document-history)
-             (st/emit! (-> (dw/remove-layout-flag :document-history)
-                           (vary-meta assoc ::ev/origin "workspace-header"))))
-
            (if (= :comments selected-drawtool)
-             (st/emit! :interrupt)
+             (do
+               (st/emit! :interrupt)
+               (when (some? comments-local)
+                 (st/emit! :interrupt)))
```
Comment (niwinz) [inline frontend/src/app/main/ui/workspace/right_header.cljs]: Do you need emit interrupt two times?

Comment (EvaMarco) [reply frontend/src/app/main/ui/workspace/right_header.cljs]: yes, one to stop the path edition other to stop the drawing mode

Comment (niwinz) [reply frontend/src/app/main/ui/workspace/right_header.cljs]: This looks like the problem is in other side. There should never be a need for two interrupts, a single interrupt should interrupt everything.

**Pull request merged**